tests/conftest.py

```diff
@@ -1,4 +1,5 @@
 """PyTest Fixtures"""
+import asyncio
```

Comment:
You are importing direct asyncio package, while in dev dependencies is pytest-asyncio plugin.

Therefore tests are still failing.

Btw, why you prefer the asyncio/aiohttp directly as a dependency and not the pytest plugins, if it is used for testing purposes only anyway? My guess would be that pytest plugin are used by many and doing IMHO the same stuff you are doing by hand - according to readme of both plugins. In this case I think is best to use standard solution of the problem instead of own implementation.

Reply:
We forgot to update dev-requirements.txt now you shouldn't have any problems with the test anymore. We have removed some useless code.

Reply:
Yeah, with 3bab585 I have passing tests, only with one warning. 

```
pytest_aiohttp\plugin.py:28: DeprecationWarning: The 'asyncio_mode' is 'legacy', switching to 'auto' for the sake of pytest-aiohttp backward compatibility. Please explicitly use 'asyncio_mode=strict' or 'asyncio_mode=auto' in pytest configuration file.
    config.issue_config_time_warning(LEGACY_MODE, stacklevel=2)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================================== 250 passed, 1 warning in 3.55s
```